doc/sphinx/language/gallina-extensions.rst

```diff
@@ -1188,29 +1155,9 @@ some of the fields and give one of its possible implementations:
 Notice that ``M`` is a correct body for the component ``M2`` since its ``T``
 component is equal ``nat`` and hence ``M1.T`` as specified.
 
-.. note::
-
-  #. Modules and module types can be nested components of each other.
-  #. One can have sections inside a module or a module type, but not a
-     module or a module type inside a section.
-  #. Commands like :cmd:`Hint` or :cmd:`Notation` can also appear inside modules and
-     module types. Note that in case of a module definition like:
-
-  ::
-
-     Module N : SIG := M.
-
-  or::
-
-    Module N : SIG. … End N.
-
-  hints and the like valid for ``N`` are not those defined in ``M``
-  (or the module body) but the ones defined in ``SIG``.
-
-
 .. _import_qualid:
 
-.. cmd:: Import @qualid
+.. cmd:: Import {+ @qualid }
```

Comment:
`Import` and `Export` could be moved to the previous section, while the printing commands could  be moved to the terminating a module section (I guess).

Reply:
Let's figure out the terminating a module section.  `Export` is accepted outside a module.  Is that actually useful or a curio?

Reply:
As I said somewhere above, as soon as you open a file, you are actually interactively defining a module.

Reply:
I moved the examples to the very end of the chapter, looks good to me.